### ISSUE TITLE
Request to add zh-HK and applying for zh-HK proofreader

### ISSUE THREAD

**Issue by lifehome** [cryptomator/ios#130]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/ios/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/ios/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

Request to add zh-HK and applying for zh-HK proofreader

### Motivation

Hi,

As the concept of information security is constantly rising up in Hong Kong, I think it is a good chance for me to translate the app into the language for the locals in HK. Thus, I am requesting the addition of zh-HK, as well as applying as a proofreader for zh-HK to expedite the translation process.

Been doing translations and localization on a few games, as well as other FOSS projects, feel free to raise out concerns!

Crowdin username: `lifehome`

-- Ivan

**Comment by overheadhunter** [comment]
I've just added `zh-HK`. Tbh, I'm not sure how to actively invite someone to a project, I think you need to request access. We're usually granting it within a day.

Wishing all the people in Hong Kong all the best these days!

**Comment by lifehome** [comment]
Hi @overheadhunter,

It's a bit weird to not see the "Join" button at the project frontpage... Gotta click inside the language to join, anyway the request is sent :)

<small>Thanks for the blessing! We do need that round the clock :sad:</small>

-- Ivan

**Comment by lifehome** [comment]
PS: I see the zh-TW is falling behind, and I can take up to it in free time for the proofreading tasks in case of it :)

-- Ivan

**Comment by overheadhunter** [comment]
Added permissions for `zh-TW` as well.

If I'm not mistaken, Hong Kong and Taiwan use traditional characters, while China mainly uses simplified Chinese. Other than that, how much do the languages have in common? Is is just a different dialect?

Due to the nature of the language codes, missing translations from `zh-HK` will fall back to `zh`, not `zh-TW`. Not sure if this can be changed somehow. As I have seen you've contributed translations to several other projects already, maybe you can share you experience with this issue?

**Comment by lifehome** [comment]
Hey there,

Thanks for the help! Indeed there are differences between `zh-hk` and `zh-tw`.

The Taiwan dialect uses a more modern and fashion tone when compared to Hong Kong, which the tone follows much of the traditional Chinese sentence structures. But speaking of the utmost main difference would be vocabularies, as only partial of them are or can be literally referenced to another.


That said, since the 90's, HK has been actively introducing the localisation from Taiwan due to the lack of manpower to translate and localise the contents and softwares, this makes the Hongkongers adapted to the use of `zh-tw` already!

In my previous contribution in the game "Keep talking and nobody explodes", I managed to coordinate with the Taiwanese, and we worked happily together to produce a `zh-tw` work by a team with both Hongkongers and Taiwanese.

Tho I rather not comment on the `zh-cn` as they are a completely separate language to me, nor I can properly proofread and adjust to the regional context accurately.

-- Ivan
